realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/UserProfileInfoTests.kt

```diff
@@ -29,7 +29,7 @@ import org.junit.runner.RunWith
 import kotlin.test.assertNull
 
 @RunWith(AndroidJUnit4::class)
-class UserProfileTests {
+class UserProfileInfoTests {
```

Comment:
I have not picked up anything about changing Profile to ProfileInfo. Is this aligned with the other SDKs?

Reply:
Sorry, I remembered wrong. JS is calling this `UserProfile` so we should rename to that as well: https://mongodb.slack.com/archives/CKW2HG3M0/p1599466293136100